discovery-infra/test_infra/helper_classes/cluster.py

```diff
@@ -149,8 +152,8 @@ def set_network_params(
             "cluster_network_cidr": env_variables['cluster_cidr'],
             "cluster_network_host_prefix": env_variables['host_prefix'],
         })
-
-        if vip_dhcp_allocation:
+        master_count = env_variables.get('master_count' , 3)
```

Comment:
Not sure why you default to 3 here and not just use what's defined in `env_variables`

Reply:
no option to set vip_dhcp_allocation in the code where this function is called from? and not to have "master_count == 1"

Reply:
OK

Reply:
@tsorya look at the code, not just the comment